radis/lbl/factory.py

```diff
@@ -1895,6 +1913,28 @@ def _generate_wavenumber_arrays(self):
 
         return
 
+    def _generate_diluent_molefraction(self, mole_fraction):
+        diluents = self.params.diluent.copy()
+
+        # Checking mole_fraction of molecule and diluent
+        total_mole_fraction = mole_fraction + sum(list(diluents.values()))
+        if total_mole_fraction > 1:
+            raise ValueError(
+                "Total molefraction = {0} of molecule and diluents greater than 1.".format(
+                    total_mole_fraction
+                )
+            )
+        elif total_mole_fraction < 1 and "air" not in diluents.keys():
```

Comment:
Good in principle, but it doesn't fit the Radis principle of "better raise an error than do something behind your back" 

I'd rather : 
- crash if total_mole_fractions are <  1  (even, if "air" is given) 
- use the error message to suggest to use a filler; for instance "Air"     (syntax `diluent='air'` or `diluent='{"species_1":mole fractoin1, "species_2" : mole_fraction2, "species_3": "fill"`  ) 
- set-up `diluent="air"` by default